### PR TITLE
added pommesmc

### PR DESCRIPTION
**Pull request by Toasty8i**
PommesMC.

## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Toasty8i** [comment]
Is this good?

File: minecraft_servers/pommesmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "gamemodes": {
+    "citybuild": {
+      "name": "CityBuild",
+      "command": "/spawn",
+      "color": "#ff00f9"
+    }
+  },
+  "discord": {
+    "server_id": 847516968478441522,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/pommesmc/manifest.json:24]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please also compress your images.

BR, Jakob

**Comment by JakobBruening** [comment]
Please fix the issue as soon as possible. Otherwise, we would close the PR.

**Pull request closed without merging**